src/videojs-contrib-hls.js

```diff
@@ -273,6 +273,7 @@ class HlsHandler extends Component {
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
 
+      _player.gapSkipper({});
```

Comment:
I would add this to the top or bottom of the constructor rather than in this if statement, as it no longer needs to be here. You also don't need to pass it an empty object.